src/core/agent/services/identifierService.ts

```diff
@@ -79,7 +79,19 @@ class IdentifierService extends AgentService {
     }
     const aid = await this.props.signifyClient
       .identifiers()
-      .get(metadata.signifyName);
+      .get(metadata.signifyName)
```

Comment:
Were the tests for all these testing that signify returned `undefined` etc? If so, they should be adjusted to throw a 404 error